### DISCUSSION TITLE
Moving "offset the camera by draging" to the front

### DISCUSSION THREAD

**Discussion by tonilxm** [flxzt/rnote#164] [Ideas]
I am figma user that use figjam feature a lot. But i recently try rnote as alternative.
Most oftenly i use "hand tool" to drag my canvas around and i find that having 2 click just to pan is just "too much work".
My suggestion is that to move the dragging tool to the front - same level as brush tool or to have option to bind it as keyboard shortcut.
Simply say the more bindings the better.

**Comment by flxzt** [comment]
I am not sure I understand what you mean by moving it to the front? As the first item in the sidebar? In any case the sidebar should remember the last used tool, so you should only need one click to access it. Additionally in the settings, you can already map one of the stylus buttons or the right mouse button to the tools page to access it quickly. (additional, configurable keyboard shortcuts are planned)
